subsys/bluetooth/host/keys.c

```diff
@@ -37,6 +39,36 @@ static struct bt_keys key_pool[CONFIG_BT_MAX_PAIRED];
 #if IS_ENABLED(CONFIG_BT_KEYS_OVERWRITE_OLDEST)
 static uint32_t aging_counter_val;
 static struct bt_keys *last_keys_updated;
+
+#if (CONFIG_BT_MAX_CONN > 1)
+static bool bt_key_is_not_in_use(uint8_t id)
```

Comment:
I'd remove the `bt_` prefix since this is a private function (seeing calls to bt_* makes me think I can find the definition in some header file).

Reply:
Done